drivers/scheduler/dcos/specs/mysql/px-mysql.json

```diff
@@ -0,0 +1,50 @@
+{
+    "id": "/pxmysql",
+    "instances": 1,
+    "cpus": 0.5,
+    "mem": 256,
+    "backoffSeconds": 1,
+    "backoffFactor": 1.15,
+    "maxLaunchDelaySeconds": 3600,
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "mysql:5.7.12",
+            "network": "BRIDGE",
+            "portMappings": [
+                {
+                    "containerPort": 3306,
+                    "hostPort": 32000,
+                    "servicePort": 10002,
+                    "protocol": "tcp",
+                    "name": "default"
+                }
+            ],
+            "privileged": false,
+            "parameters": [
+                {
+                    "key": "volume-driver",
+                    "value": "pxd"
```

Comment:
These specs should be templated out. The driver field should be populated depending on which volume driver is being tested. It looks like it is the same for the k8s specs. So they can be fixed together.

Reply:
Created an issue to track this: https://github.com/portworx/torpedo/issues/70